ci/builders/mac_host_engine.json

```diff
@@ -17,7 +17,8 @@
             ],
             "drone_dimensions": [
                 "device_type=none",
-                "os=Mac-12"
+                "os=Mac-12",
+		"cpu=x64"
```

Comment:
shall this be `cpu=x86`?

Reply:
We don't have any x86 machines, I believe if x86 is there is for backward compatibility.